docs/DeveloperGuide.md

```diff
@@ -1,34 +1,152 @@
+    * month - if user does not input month, it uses computer's current month
+    * startingDay - which day of the week the first day of the months begins on (0-6, where 0 is Monday)
+    * totalDays - how many days in that month
+    * totalWeeks - number of weeks of the month
+  
+1. The `calendar` command calls on TaskList#findTaskDate() for each day of the month to generate the task listing for a particular day. Only the task event and description is added to calendar view. 
+1. The calendar generated is then stored in a string and UI#showUserMesssage() is called to display the calendar. 
+
+The class diagram below shows the relationships between the different classes required by the `calendar` feature.
+![Sequence diagram for CalendarCommand](images/CalendarCommand_class.jpg)
+
+
+The following diagram summarises what happens when a user executes a new `calendar` command:
+![Sequence diagram for CalendarCommand](images/CalendarCommand_sequence.jpg)
+
+
+##### 3.1.2.2 Design Considerations
+1. Aspect: Obtaining information required for generating monthly view
+    
+    * Alternative 1: (current choice) Algorithm to deduce how many weeks in month, which day a date is, how many days in that month
+        * Pros: Will use less memory, requiring only one starting date we are able to derive any other dates. No need to worry about changing template for the new year
+        * Cons: New developers will take additional effort to understand how the algorithm works.
+   
+    * Alternative 2: Hardcoded information (constant variables to tell days in month/how many weeks)
+        * Pros: Easy maintenance, tedious but easily calculated with web applications.
+        * Cons: Tedious and not sustainable, constant updates have to be done to edit the fields for a new year
+
+1. Aspect: Which month to use
+   
+    * User might want to know schedule for other months, but might also want a quick view of current month
+    * Solution is to set a default month (taken from computer) and also allow input for preferred month.
+ 
+1. Aspect: generation of monthly details 
+    
+    * Calculation of details are shifted from the command to a separate class. 
+    This is to enable easier maintenance for methods relating to calendar features.
+    
+## 4. Documentation
+
+## 5. Testing
+There are two ways to do the test.
+Method 1: Using IntelliJ JUnit test runner
+To run all tests, right-click on the src/test/java folder and choose Run `All Tests`
+To run a subset of tests, you can right-click on a test package, test class, or a test and choose Run  `ABC`
+
+Method 2: Using Gradle
+Open a console and run the command gradlew clean test (Mac/Linux: ./gradlew clean test)
+## 6. Dev Ops
+### 6.1 Appendix A:Product Scope
+Target User profile:
+1. University students.
+2. Prefer desktop apps.
+3. Can type fast
+
+### 6.2 Appendix B:Value proposition
+It solves:
+1. Daily tasks arrangement
+2. Class schedule arrangement
+3. Event alert
+
+## 6.3 Appendix C:User Stories
```

Comment:
I think there might be some formatting issue for this table. This is what is displayed on my screen

![image](https://user-images.githubusercontent.com/59786675/77991480-8f7df600-7356-11ea-97be-753c2fcb54fb.png)